docker-bake.hcl

```
@@ -31,6 +31,13 @@ group "linux-s390x" {
   ]
 }
 
+group "linux-ppc64le" {
+  targets = [
+    "debian_jdk11",
+    "rhel_ubi9_jdk17",
```

is there a reason you need it on rhel? I would suggest just debian 11 and 17
```suggestion
    "debian_jdk11",
    "debian_jdk17",
```

---

Hello, Tim. Yes, we have at least one customer actively using the UBI controller image. We can include all three images in the build:

debian_jdk11
debian_jdk17
rhel_ubi9_jdk17